app/config/locale/translations/uk.json

```diff
@@ -1,32 +1,32 @@
 {
-    "settings.inspire": "\"The art of being wise is the art of knowing what to overlook.\"",
+    "settings.inspire": "\"Мистецтво бути мудрим - це мистецтво знати, чим можна знехтувати\"",
     "settings.locale": "uk",
     "settings.direction": "ltr",
     "emails.sender": "Команда %s",
-    "emails.verification.subject": "",
-    "emails.verification.hello": "",
-    "emails.verification.body": "",
-    "emails.verification.footer": "",
-    "emails.verification.thanks": "",
-    "emails.verification.signature": "",
-    "emails.magicSession.subject": "",
-    "emails.magicSession.hello": "",
-    "emails.magicSession.body": "",
-    "emails.magicSession.footer": "",
-    "emails.magicSession.thanks": "",
-    "emails.magicSession.signature": "",
-    "emails.recovery.subject": "",
-    "emails.recovery.hello": "",
-    "emails.recovery.body": "",
-    "emails.recovery.footer": "",
-    "emails.recovery.thanks": "",
-    "emails.recovery.signature": "",
-    "emails.invitation.subject": "",
-    "emails.invitation.hello": "",
-    "emails.invitation.body": "",
-    "emails.invitation.footer": "",
-    "emails.invitation.thanks": "",
-    "emails.invitation.signature": "",
+    "emails.verification.subject": "Верифікація акаунта",
+    "emails.verification.hello": "Вітаємо, {{name}}",
+    "emails.verification.body": "Перейдіть за цим посиланням, щоб підтвердити свою електронну адресу.",
+    "emails.verification.footer": "Якщо ви не запитували підтвердження цієї адреси, ви можете ігнорувати це повідомлення.",
+    "emails.verification.thanks": "Дякуємо",
+    "emails.verification.signature": "команда {{project}}",
+    "emails.magicSession.subject": "Логін",
+    "emails.magicSession.hello": "Вітаємо,",
+    "emails.magicSession.body": "Перейдіть за цим посиланням, щоб увійти.",
+    "emails.magicSession.footer": "Якщо ви не просили увійти за допомогою цієї електронної пошти, ви можете ігнорувати це повідомлення.",
+    "emails.magicSession.thanks": "Дякуємо",
+    "emails.magicSession.signature": "команда {{project}}",
+    "emails.recovery.subject": "Скидання пароля",
+    "emails.recovery.hello": "Вітаємо, {{name}}",
+    "emails.recovery.body": "Перейдіть за цим посиланням для того щоб скинути свій пароль для проекту {{project}}",
+    "emails.recovery.footer": "Якщо ви не запитували скидання паролю, проігноруйте це повідомлення.",
+    "emails.recovery.thanks": "Дякуємо",
+    "emails.recovery.signature": "команда {{project}}",
+    "emails.invitation.subject": "Запрошення до %s Команди у %s",
+    "emails.invitation.hello": "Вітаємо",
+    "emails.invitation.body": "Цей лист був надісланий вам тому що <b>{{owner}}</b> запрошує вас стати членом команди <b>{{team}}</b> у проекті {{project}}.",
```

Comment:
Using `<b>` tags is a great addition, but it may not be recommended since our baseline [English translation](https://github.com/appwrite/appwrite/blob/master/app/config/locale/translations/en.json#L26) and others do not use bold lettering. We can do this in the future but let's keep it simple for the sake of this PR.